### ISSUE TITLE
How to finetune on custom loss function?

### ISSUE THREAD

**Issue by gopal86** [amazon-science/chronos-forecasting#29]
Hi, I wanted to fine tune the model on my own dataset however with my own custom loss. Could you give an example on how to do that? It would be very helpful for my research purpose! I am unclear on how to do that on your model  

Thanks Gopal

**Comment by abdulfatir** [comment]
Thanks for your interest @gopal86, please check #12.
